js/assessments/sentenceLengthInTextAssessment.js

```diff
@@ -13,25 +14,26 @@ var calculateSentenceLengthResult = function( sentences, i18n ) {
 	var maximumPercentage = 25;
 
 	var tooLong = countTooLongSentences( sentences, recommendedValue );
-	var percentage = ( tooLong / sentences.length ) * 100;
+	var percentage = fixFloatingPoint( ( tooLong / sentences.length ) * 100 );
 
 	var score = calculateTooLongSentences( percentage );
 
 	if ( score >= 7 ) {
 		return{
 			score: score,
 
-			// translators: %1$s expands to number of sentences.
-			text: i18n.sprintf( i18n.dgettext( "js-text-analysis", "%1$s of the sentences contain more than 20 words, " +
-				"which is within the recommended range." ), percentage + "%" )
+			// Translators: %1$s expands to number of sentences.
```

Comment:
Please add translator comments for %3$s, also shouldn't this be %3$d, since we only put a digit in there.